packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -287,6 +287,23 @@ storiesOf('TableView', module)
       </TableView>
     )
   )
+  .add(
+    'dynamic with disabled, multiple selection, highlight',
+    () => (
+      <TableView disabledKeys={['Foo 1', 'Foo 3']} aria-label="TableView with dynamic contents" selectionStyle="highlight" selectionMode="multiple" width={300} height={200} onSelectionChange={s => onSelectionChange([...s])}>
```

Comment:
I wish there was a way to surface which keys are disabled without having to remember or check the code.